msteams-platform/graph-api/meeting-transcripts/get-transcripts-api.md

```diff
@@ -441,32 +447,71 @@ To obtain meeting ID and organizer ID from tenant-level notification:
+
+        - The meeting ID is included as the value for `onlineMeetings`: *MSoxNGI3NzlhZS1jYjY0LTQ3ZTctYTUxMi01MmZkNTBhNDE1NGQqMCoqMTk6bW
+    VldGluZ19ObVUwTlRreFl6TXRNMlkyTXkwME56UmxMV0ZtTjJZdE5URmlNR001T1dNM
+    1pqWTJAdGhyZWFkLnYy*
+        - The organizer ID is *14b779ae-cb64-47e7-a512-52fd50a4154d*
+
+        The response payload contains the transaction ID for the meeting ID and organizer ID.
+
+        ```json
+        {
+
+            "@odata.context": "https://graph.microsoft.com/v1.0/$metadata#users('14b779ae-cb64-47e7-a512-52fd50a4154d')/onlineMeetings('MSoxNGI3NzlhZS1jYjY0LTQ3ZTctYTUxMi01MmZkNTBhNDE1NGQqMCoqMTk6bWVldGluZ19ObVUwTlRreFl6TXRNMlkyTXkwME56UmxMV0ZtTjJZdE5URmlNR001T1dNM1pqWTJAdGhyZWFkLnYy')/transcripts",
+        
+            "@odata.count": 1,
+        
+            "value": [
+        
+                {
+        
+                    "id": "MSMjMCMjMDEyNjJmNjgtOTc2Zi00MzIxLTlhNDQtYThmMmY4ZjQ1ZjVh",
+        
+                    "createdDateTime": "2022-04-14T11:34:39.5662792Z"
+        
+                }
+        
+            ]
+        
+        }
+        ```
+
+    1. **Access and get meeting transcript based on the transcript ID**:
+
     Use the following example to request the transcripts for a specific meeting in the `.vtt` format:
 
     ```http
-    GET https://graph.microsoft.com/beta/users('14b779ae-cb64-47e7-a512-52fd50a4154d')/onlineMeetings('MSoxNGI3NzlhZS1jYjY0LTQ3ZTctYTUxMi01MmZkNTBhNDE1NGQqMCoqMTk6bWVldGluZ19ObVUwTlRreFl6TXRNMlkyTXkwME56UmxMV0ZtTjJZdE5URmlNR001T1dNM1pqWTJAdGhyZWFkLnYy')/transcripts('MSMjMCMjMDEyNjJmNjgtOTc2Zi00MzIxLTlhNDQtYThmMmY4ZjQ1ZjVh')/content?$format=text/vtt
+    GET https://graph.microsoft.com/v1.0/users('14b779ae-cb64-47e7-a512-52fd50a4154d')/onlineMeetings('MSoxNGI3NzlhZS1jYjY0LTQ3ZTctYTUxMi01MmZkNTBhNDE1NGQqMCoqMTk6bWVldGluZ19ObVUwTlRreFl6TXRNMlkyTXkwME56UmxMV0ZtTjJZdE5URmlNR001T1dNM1pqWTJAdGhyZWFkLnYy')/transcripts('MSMjMCMjMDEyNjJmNjgtOTc2Zi00MzIxLTlhNDQtYThmMmY4ZjQ1ZjVh')/content?$format=text/vtt
     ```
 
-    In this example:
-    - The meeting ID is included as the value for `onlineMeetings`: *MSoxNGI3NzlhZS1jYjY0LTQ3ZTctYTUxMi01MmZkNTBhNDE1NGQqMCoqMTk6bW
-VldGluZ19ObVUwTlRreFl6TXRNMlkyTXkwME56UmxMV0ZtTjJZdE5URmlNR001T1dNM
-1pqWTJAdGhyZWFkLnYy*
-    - The organizer ID is *14b779ae-cb64-47e7-a512-52fd50a4154d*
-
     The response payload will contain the transcripts in `.vtt` format.
 
 ### Use Bot Framework to get meeting ID and organizer ID
 
 Your app can use the Bot Framework for obtaining meeting ID and organizer ID.
 
 > [!NOTE]
-> The bot can receive meeting start or end events automatically from all the meetings created in all the channels by adding `ChannelMeeting.ReadBasic.Group` to manifest for meeting-specific RSC application permissions.
+> The bot can receive meeting start or end events automatically from all the scheduled online meetings. Add / manifest configuration / to manifest for granting meeting-specific RSC application permissions.
```

Comment:
Is this WIP?
I wasn't sure what 'Add / manifest configuration / to manifest' meant

Reply:
This was waiting on response for the change in the permission name for online meetings. I'd left a placeholder for correction. It's corrected now after response from Wajeed.